config/locales/en.yml

```diff
@@ -21,3 +21,16 @@
 
 en:
   hello: "Hello world"
+
+  activerecord:
+    attributes:
+      user:
+        password: "Password"
+    errors:
+      models:
+        user:
+          attributes:
+            password:
+              blank: "is blank!"
+            password_confirmation:
+              blank: "does not match confirmation!"
```

Comment:
ahh. please coordinate with @tudatn about tests 😸

Reply:
not a test--this is just a custom error message.